api/v1alpha1/driver_types.go

```diff
@@ -213,6 +213,11 @@ type ControllerPluginSpec struct {
 	// Embedded common pods spec
 	PodCommonSpec `json:",inline"`
 
+	// DeploymentStrategy describes how to replace existing pods with new ones
+	// Default value is RollingUpdate with MaxUnavailable and MaxSurege as 25% (kubernetes default)
+	//+kubebuilder:validation:Optional
+	DeploymentStrategy *appsv1.DeploymentStrategy `json:"deploymentStrategy,omitempty"`
```

Comment:
If we can predict that users might want to also have access to the  MaxUnavailable and MaxSurege params then this configuration needs to become an object

Reply:
didn't get you, those options are already part of the `appsv1.DeploymentStrategy` struct.

Reply:
Sorry, I got confused between DeploymentStategy and the underlying RoolingUpdate field.